carbonmark-api/src/routes/projects/get.ts

```diff
@@ -44,19 +46,20 @@ const handler = (fastify: FastifyInstance) =>
     //Transform the list params (category, country etc) provided so as to be an array of strings
     const args = mapValues(omit(request.query, "search"), split(","));
     //Get the default args to return all results unless specified
-    const defaultArgs = await getDefaultQueryArgs(fastify);
-
-    //Build our query overriding default values
-    const query: FindProjectsQueryVariables = {
-      ...defaultArgs,
-      ...args,
-      search: request.query.search ?? "",
-    };
+    const allOptions = await getDefaultQueryArgs(fastify);
 
     const [marketplaceProjectsData, poolProjectsData, cmsProjects, poolPrices] =
       await Promise.all([
-        gqlSdk.marketplace.findProjects(query),
-        gqlSdk.offsets.findCarbonOffsets(query),
+        gqlSdk.marketplace.findProjects({
+          vintage: args.vintage ?? allOptions.vintage,
```

Comment:
any possibility these can be empty strings? if yes then use `||` instead of `??`